tests/Agency/CleanUpLostCollectionTest.cpp

```diff
@@ -125,54 +132,60 @@ SECTION("clean up a lost collection when the leader is failed") {
     //        old: /arango/Supervision/Health/leader/Status == "FAILED"
 
     auto const& trxs = q->slice();
-    REQUIRE(trxs.length() == 1);
+    EXPECT_TRUE(trxs.length() == 1);
```

Comment:
`REQUIRE` in catch == `ASSERT_TRUE` in gtest

Reply:
True, but the `ASSERT_` methods issue a `return;` and so cannot be used in methods with a non-void return type.

Reply:
got it